rules/REQUEST-901-INITIALIZATION.conf

```diff
@@ -8,6 +8,31 @@
 # ---------------------------------------------------------------
 
 #
+# -=[ Initialization / General Setup ]=-
+#
+# Some constructs or individual rules will fail if certain parameters
+# are not set in the setup.conf file. The following rules will catch
+# these cases and assign sane default values.
+#
+
+# Default Paranoia Level (should be defined in rule 900023 in setup.conf)
+SecRule &TX:paranoia_level "@eq 0" \
+    "id:901100,\
+    phase:1,\
+    pass,\
+    nolog,\
+    setvar:tx.paranoia_level=1"
+
+# Default Sampling Percentage (should be defined in rule 900025 in setup.conf)
+SecRule &TX:sampling_percentage "@eq 0" \
```

Comment:
I noticed that you added this rule instead of relying on the value in setup.conf to be 100 by default. Any reason?